packages/fxa-settings/src/pages/Pair/Unsupported/en.ftl

```diff
@@ -0,0 +1,4 @@
+## PairUnsupported - a view which is shown when the user tries to scan the pairing QR code any way other than through a Firefox app
+
+unsupported-header = Pair using an app
+unsupported-message = Did you use the system camera? You must pair from within a { -brand-firefox } app.
```

Comment:
Minor nit: Not blocking but the IDs `unsupported-header` might be a bit generic. I doubt the risk for ID duplication is that high, but maybe `pair-unsupported-header` / `-message` might be better?

Reply:
Ooh that's a great idea, thank you!